src/hooks/sdk/docs/useSelectVideoInputDevice.stories.mdx

```diff
@@ -14,6 +14,11 @@ You can access the available video input device IDs using the `useVideoInputs` h
 (deviceId: string) => Promise<void>;
 ```
 
+Use the returned function from hook only when you want to start a video or the video is already started.
+If used earlier and you select a valid video input with access to your local camera, the webcam LED will turn on.
+By default, we internally just select the first device listed by `useVideoInputs` hook, but we do not actually choose it by calling Amazon Chime SDK for JavaScript's `chooseVideoInputDevice` API.
```

Comment:
I am a little confused, here says "we do not actually choose it by calling Amazon Chime SDK for JavaScript's `chooseVideoInputDevice` API." but in the part below, it says "it internally calls the `chooseVideoInputDevice` API from the Amazon Chime SDK for JavaScript."

Reply:
Have removed this, thanks it sounds confusing thanks for the catching it.